packages/template-retail-react-app/app/components/search/partials/suggestions.jsx

```diff
@@ -23,11 +30,50 @@ const Suggestions = ({suggestions, closeAndNavigate}) => {
                         key={idx}
                         marginTop={0}
                         variant="menu-link"
+                        style={{justifyContent: 'flex-start', padding: '8px 12px'}}
                     >
-                        <Text
-                            fontWeight="400"
-                            dangerouslySetInnerHTML={{__html: suggestion.name}}
-                        />
+                        <Flex align="center">
+                            {/* Reserve space for image for all, but only render if present */}
+                            <Box
+                                width="40px"
+                                height="40px"
+                                marginRight="16px"
```

Comment:
How do these hard coded px numbers look in different screen sizes & resolutions?